catalog/task/update-ocp-tag/0.1/update-ocp-tag.yaml

```diff
@@ -0,0 +1,57 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: update-ocp-tag
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: release
+spec:
+  description: >-
+    Tekton task to update pull-spec tag with
+    valid OCP version from get-ocp-version task.
+  params:
+    - name: fromIndex
+      description: The source Index image (catalog of catalogs) FBC fragment
+      type: string
+    - name: targetIndex
+      description: Index image (catalog of catalogs) the FBC fragment will be added to
+      type: string
+    - name: binaryImage
+      description: OCP binary image to be baked into the index image
+      type: string
+    - name: ocpVersion
+      description: Stored OCP version from get-ocp-version task result
+      type: string
+  results:
+    - name: updated-fromIndex
+      description: Store OCP version number from given Image
```

Comment:
What does this mean? Does it store the ocp version? In that case just `OCP version number from given Image` would be more suitable. This applies to the others as well.

Reply:
But looking at what it does, it's exactly the same as the source params, only the tag is modified to include the new ocpVersion, right? So the description should say that.